pkg/auth/config/secret_data.go

```diff
@@ -0,0 +1,195 @@
+package config
+
+import (
+	"github.com/skygeario/skygear-server/pkg/validation"
+)
+
+var _ = SecretConfigSchema.Add("DatabaseCredentials", `
+{
+	"type": "object",
+	"properties": {
+		"database_url": { "type": "string" },
+		"database_schema": { "type": "string" }
+	},
+	"required": ["database_url"]
+}
+`)
+
+type DatabaseCredentials struct {
+	DatabaseURL    string `json:"database_url,omitempty"`
+	DatabaseSchema string `json:"database_schema,omitempty"`
+}
+
+func (c *DatabaseCredentials) SetDefaults() {
+	if c.DatabaseSchema == "" {
+		c.DatabaseSchema = "public"
```

Comment:
Let's use `app`?

Reply:
`public` schema is the default schema of Postgres, so developer does not need to create a new schema for quick start.